scaleway/resource_iot_hub.go

```diff
@@ -226,6 +239,17 @@ func resourceScalewayIotHubRead(ctx context.Context, d *schema.ResourceData, met
 	_ = d.Set("disable_events", response.DisableEvents)
 	_ = d.Set("events_topic_prefix", response.EventsTopicPrefix)
 	_ = d.Set("device_auto_provisioning", response.EnableDeviceAutoProvisioning)
+	_ = d.Set("mqtt_ca_url", computeIotHubCaURL(response.ProductPlan, region))
+	mqttURL := d.Get("mqtt_ca_url")
+	mqttCa, err := computeIotHubMQTTCa(ctx, fmt.Sprintf("%v", mqttURL), meta)
+	if err != nil {
+		_ = diag.Diagnostic{
+			Severity:      diag.Warning,
+			AttributePath: cty.GetAttrPath("mqtt_ca"),
+			Summary:       err.Error(),
```

Comment:
Diagnostic allows a Summary and details.
Summary could be used to summarize the issue (ex: Failed to fetch io hub mqtt certificate).
And details can contain the real error that may be an error from go http client.